Sources/XMLParser/Conversions.swift

```diff
@@ -34,18 +34,18 @@ extension Conversions {
         }
     }
 
-    struct UnpackXMLElementTuple: Conversion {
+    struct UnpackXMLElement: Conversion {
         @inlinable
         init() { }
-        
+
         @inlinable
-        func apply(_ input: ((String, OrderedDictionary<String, String>), [XML.Node], String)) -> (String, OrderedDictionary<String, String>, [XML.Node]) {
-            (input.0.0, input.0.1, input.1)
+        func apply(_ input: (String, OrderedDictionary<String, String>, [XML.Node], String)) -> (String, OrderedDictionary<String, String>, [XML.Node]) {
```

Comment:
Can you elaborate why this change of signature is needed?

Reply:
(Why it's now possible to do without the previous nested tuples)

Reply:
tbh, I am not sure. I fixed based on the type mismatches which posed up.

That said, there hv been some changes in the ParserBuilders (https://github.com/pointfreeco/swift-parsing/discussions/290), which might hint to the reason.

In particular, see how in the PrefixTest we now have to specifically add the Output as second parameter (as a Tuple) to the ParserPrinter, whereas before, it was probably inferred as such:
https://github.com/pointfreeco/swift-parsing/commit/c6e2241daa46e5c6e5027a93b161bca6ba692bcc

Reply:
Seems like tuples are now unpacked automatically. Which is pretty nice